stock_picking_mgmt_weight/report/stock_picking_tag_template.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <template id="label_transfer_template_view_pdf">
+        <t t-call="web.basic_layout">
+            <div class="page" style="display: -webkit-box; display: -webkit-flex; display: flex; -webkit-box-pack: center; justify-content: center; align-items: center; -webkit-box-align: center;">
+                <t t-foreach="docs" t-as="picking">
+                    <t t-foreach="picking.move_lines" t-as="move">
+                        <div class="w-100" style="page-break-before: always;">
+                            <div class="w-100 text-right mt-1 mr-1">
+                            </div>
+                            <div class="w-100 text-right mt-1 mr-1">
+                                <span>P21/6986</span>
```

Comment:
At the moment I have put the content in invisible since if I remove the content and only leave the container, the next container raises it to the initial position... I am looking for a solution to this problem